cats/__init__.py

```diff
@@ -290,6 +289,29 @@ def schedule_at(
         return f"Scheduling with at failed with code {e.returncode}, see output below:\n{e.output}"
 
 
+def schedule_sbatch(output: CATSOutput, args: list[str]) -> Optional[str]:
```

Comment:
There is an interesting difference between this signature and the schedule_at signature - for at we allow (in the code, but the user cannot access) alternative choices to the default "at" command to be passed in. Here you don't, "sbatch" is hard coded below. I think this is fine, but it's probably worth a comment. (TBH, I'm not clear why we go through the Popen and piping process for at either - but do remember that this was a PITA to get working).

Reply:
Not sure why we allow at_command to be overridden, AFAIK all POSIX systems should have the same name for at. I will remove the at_command from the schedule_at() call. As for the piping for at, that is because at accepts the command name through stdin.

Reply:
I think it gets used in the tests - to see what happens when at is broken - but there are other ways of doing that I guess.